src/components/AssetTree/stories/basic.story.mdx

```diff
@@ -0,0 +1,63 @@
+import { Meta, Story, Preview, Props, Source } from '@storybook/addon-docs/blocks';
+import { AssetTree } from '../AssetTree';
+import { decorators } from './helper';
+
+<Meta title="Assets|AssetTree" decorators={decorators} />
+
+# Asset Tree
+
+Visualize parent-child relationship of assets in a tree structure. It will initially fetch all available root assets from the SDK. When a node is expanded, the children will be fetched dynamically
+
+**NOTE:** The component should have `ClientSDKProvider` as a parent component in react component tree.
+
+## Basic usage:
+
+Imports:
+
+```jsx
+import 'antd/dist/antd.css';
+
+import React from 'react';
+import { AssetTree } from '@cognite/gearbox';
+
+```
+
+<Preview>
+  <Story name="Basic Usage">
+    <AssetTree/>
+  </Story>
+</Preview>
+
+## Available props
+
+<Props of={AssetTree} />
```

Comment:
Please, redo this in another way. It's possible to show props properly for this component

Reply:
@mrtraser What did you mean by _another way_? edit the assetTree component?

Reply:
check `AssetBreadcrumb` example